app/konnect/index.md

```diff
@@ -79,7 +79,7 @@ across the organization’s applications.
 collaborate and manage their own set of data plane nodes and services without
 the risk of impacting other teams and projects. Gateway Manager instantly
 provisions a hosted {{site.base_gateway}} control plane and supports securely
-attaching data plane nodes from your cloud or hybrid environments.
+attaching data plane nodes from your fully-managed cloud, or self-managed cloud and hybrid environments.
```

Comment:
```suggestion
attaching self-hosted, or fully-managed data plane nodes and hybrid environments.
```

Reply:
Suggestion off of Angel's suggestion, as the "hybrid environments" portion of the sentence wasn't in the right place.

```suggestion
attaching self-hosted nodes running in cloud or hybrid environments, or using fully-managed data plane nodes.
```
I'm still not 100% on this phrasing, but the point I'm trying to get to it - self-hosted is what runs anywhere, fully-managed doesn't have that control.